tests/test_other.py

```diff
@@ -10307,10 +10307,16 @@ def test_threadprofiler_closure(self):
   def test_syslog(self):
     self.do_other_test('test_syslog.c')
 
-  def test_split_module(self):
+  @parameterized({
+    'default': (False,),
```

Comment:
You can just use the empty string here rather than `"default"`. 

That will result in two tests:

`test_split_module`

and

`test_split_module_custom`